packages/validation/frontend/sirius-components-validation/src/ValidationView.tsx

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
```

Comment:
![ValidationView](https://github.com/user-attachments/assets/a3f9b08a-8717-47b1-9a23-ead2f9848927)
I got a weird behavior when opening this view